### ISSUE TITLE
ToDo for next steps

### ISSUE THREAD

**Issue by choldgraf** [neuro-data-science/neuro_data_science#14]
These are all based off of the conversation we recently had at NeuroHackWeek 2017. There was a lot of interest in this and hopefully we can use it to move forward.  @satra feel free to jump in and make edits/suggestions/etc

## Principles and governance
* What are minimal standards to be met for new contributions to be accepted?
  * E.g., needs exercises, needs tests, must run from start-to-finish, etc
* How to determine the scope that this material will cover?
* How to specify dependencies (e.g. requirements.txt, environment.yml)
* Decide on how to incorporate new languages into the process (e.g. Matlab, R, or shell software)
* Combine curation and adding new content - reuse wherever possible.

## Longer-term
* Define pathways through modules of material
  * Based off of skill level
  * Based off of the type of user / background level
  * Based off of the thing you want to learn?

## Next steps
  * Clean up the content that's in here now
  * Determine the main content sections we want to cover
  * Identify new content that needs to be added
  * Define the structure of this repository

Taken from this whiteboarding session: https://files.slack.com/files-tmb/T536K9PEK-F6Z81238B-c3d7e7a066/img_20170906_150256_1024.jpg

![image](https://user-images.githubusercontent.com/184063/30231657-cdc45806-94a0-11e7-8278-cdd542507cb4.png)

**Comment by arokem** [comment]
One random thought here: to be effective, maybe a needs assessment would be useful. What materials are actually needed? What would get used by instructors and students if it were cleaner/nicer/more comprehensive? I am not sure what the best methodology here would be (e.g., surveys, targeted interviews, breakout sessions at conferences, divine inspiration...), but it would be good to be systematic about the second point in "next steps".

**Comment by choldgraf** [comment]
yeah I agree - a short-term approach is to just define content that we "need" based on our previous experience as instructors...but I agree a better long-term approach is to be systematic about our target audience of users...
